src/main/java/Ui.java

```diff
@@ -0,0 +1,44 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Ui {
+    protected static ArrayList<String> commandList = new ArrayList<>(Arrays.asList
+            ("todo", "deadline", "event", "mark", "unmark", "list", "bye", "delete"));
+    public void run(TaskList records) {
+        greet();
+        Scanner userInput = new Scanner(System.in);
+        Parser p = new Parser(records);
+        while (true) {
+            try {
+                String input = userInput.nextLine();
+                //Check for empty input (pressing enter)
+                if (input == "") {
+                    throw new EmptyInputException();
+                }
+                String[] inputArr = input.split(" ", 2);
+                //Check for invalid commands (first word)
+                boolean validCommand = commandList.contains(inputArr[0]);
```

Comment:
Can edit boolean variable to sound more like boolean variable. Eg isValidcomment